sources/host-ctr/cmd/host-ctr/main.go

```diff
@@ -1184,3 +1201,15 @@ func withUnmaskedPaths(unmaskPaths []string) oci.SpecOpts {
 		return nil
 	}
 }
+
+// Convert labels to map[string]string for containerd.WithPullLabels.
+// Labels are in the format of "key=value,key2=value2".
+func convertLabels(labels string) map[string]string {
```

Comment:
I'd like to see some unit tests for this since we are taking input from the cli which should have some reasonable validation that we are transforming this correctly into the right map.